bin/flutter

```diff
@@ -41,11 +41,11 @@ function _rmlock () {
 
 function retry_upgrade {
   local total_tries="10"
-  local remaining_tries=$(($total_tries - 1))
-  while [[ "$remaining_tries" > 0 ]]; do
+  local remaining_tries=$((total_tries - 1))
+  while [[ "$remaining_tries" -gt 0 ]]; do
```

Comment:
So apparently `>` does a string character code comparison, whereas `-gt` will do a numeric value comparison. Now the string "0" is still less than all other numeric strings, but better safe than sorry.